<!--
  ~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
  ~
  ~ WSO2 LLC. licenses this file to you under the Apache License,
  ~ Version 2.0 (the "License"); you may not use this file except
  ~ in compliance with the License.
  ~ You may obtain a copy of the License at
  ~
  ~ http://www.apache.org/licenses/LICENSE-2.0
  ~
  ~ Unless required by applicable law or agreed to in writing,
  ~ software distributed under the License is distributed on an
  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
  ~ KIND, either express or implied.  See the License for the
  ~ specific language governing permissions and limitations
  ~ under the License.
  -->

<p>Don’t use one letter variable names (except ‘e’ for exceptions, ‘i, j’ in for loops)</p>

<h2>Noncompliant Code Example</h2>
<pre>
int i;
String l;
boolean p;
float k;
char a;
</pre>

<h2>Compliant Solution</h2>
<pre>
int number;
String name;
</pre>
